pkg/cmd/cli/cmd/create/user.go

```diff
@@ -0,0 +1,122 @@
+package create
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/api/meta"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	userapi "github.com/openshift/origin/pkg/user/api"
+)
+
+const (
+	UserRecommendedName = "user"
+
+	userLong = `
+Create a user.`
```

Comment:
Expound on the help to indicate when I'd want to do use this (and the other) commands.  This is only likely to be used if my master-config is set to avoid any auto-provisioning, right?  We don't want people thinking they should do this.